integration-tests/testutils/report/parser.go

```diff
@@ -95,12 +95,12 @@ func (fr *SubunitV2ParserReporter) Write(data []byte) (int, error) {
 
 	if matches := announceRegexp.FindStringSubmatch(sdata); len(matches) == 2 {
 		testID := matches[1]
-		if isTest(testID) {
+		if isTest(testID) && !common.IsInRebootProcess() {
 			err = fr.statuser.Status(subunit.Event{TestID: matches[1], Status: "exists"})
 		}
 	} else if matches := successRegexp.FindStringSubmatch(sdata); len(matches) == 2 {
```

Comment:
After your comment, and then after reading this, I'm now concerned with how fragile the framework is starting to look. These expressions will break when trivial things change in the output of the testing tool, and it looks like it'll be a nightmare to find out why something like that has happened.

Can we take a serious look at that concern sometime soon, and check what can be done to start simplifying the framework so it's easier to see what's going on and harder to break it?

Not about this branch, though.